Makefile

```diff
@@ -18,3 +18,25 @@ quality:
 # Evaluation
 
 evaluate:
+	$(eval PARALLEL_ARGS := $(if $(PARALLEL),$(shell \
+		if [ "$(PARALLEL)" = "data" ]; then \
+			echo "data_parallel_size=$(NUM_GPUS)"; \
+		elif [ "$(PARALLEL)" = "tensor" ]; then \
+			echo "tensor_parallel_size=$(NUM_GPUS)"; \
+		fi \
+	),))
+	$(if $(filter tensor,$(PARALLEL)),export VLLM_WORKER_MULTIPROC_METHOD=spawn &&,) \
+	MODEL_ARGS="pretrained=$(MODEL),dtype=float16,$(PARALLEL_ARGS),max_model_length=32768,gpu_memory_utilisation=0.8" && \
+	lighteval vllm $$MODEL_ARGS "custom|$(TASK)|0|0" \
+		--custom-tasks src/open_r1/evaluate.py \
+		--use-chat-template \
+		--system-prompt="Please reason step by step, and put your final answer within \boxed{}." \
+		--output-dir data/evals/$(MODEL)
+
+# Example usage:
+# Single GPU:
+#   make evaluate MODEL=deepseek-ai/DeepSeek-R1-Distill-Qwen-1.5B TASK=aime24
+# Data parallel:
+#   make evaluate MODEL=deepseek-ai/DeepSeek-R1-Distill-Qwen-1.5B TASK=aime24 PARALLEL=data NUM_GPUS=8
+# Tensor parallel:
+#   make evaluate MODEL=deepseek-ai/DeepSeek-R1-Distill-Qwen-1.5B PARALLEL=tensor NUM_GPUS=8
```

Comment:
```suggestion
```

Can you make another PR for this please?

Reply:
Of course! I did open a new PR but didn't realize I had included the second commit also here 🙏